src/components/swap/SwapBuyFiatButton.test.tsx

```diff
@@ -45,6 +47,7 @@ describe('SwapBuyFiatButton.tsx', () => {
   let useOpenModal: jest.Mock<any, any>
 
   beforeAll(() => {
+    store.dispatch(setOriginCountry('US'))
```

Comment:
Can you also add tests for when origin country is `GB` and `null`?

Reply:
I'm adding tests when I implement caching tomorrow because it's just annoying to test with the opacity stuff right now and when caching is implemented it will be easier